tests/plugins/pastefromword/generated/_helpers/pfwTools.js

```diff
@@ -90,14 +90,15 @@
 			} ),
 
 			// Firefox adds quotes around all fonts in font-family.
+			// Safari replaces `&guot;` with `'` in quoted font names.
```

Comment:
The similar (reversed) situation happens in Chrome, where `'Arial'` is replaced (when styles are parsed by the browser) with `"Arial"`.